common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/ExecutorDiskUtils.java

```diff
@@ -38,9 +38,9 @@ public static File getFile(String[] localDirs, int subDirsPerLocalDir, String fi
     // Unfortunately, we cannot just call the normalization code that java.io.File
     // uses, since it is in the package-private class java.io.FileSystem.
     // So we are creating a File just to get the normalized path back to intern it.
-    // Finally a new File is built and returned with this interned normalized path.
+    // We return this interned normalized path.
     final String normalizedInternedPath = new File(notNormalizedPath).getPath().intern();
-    return new File(normalizedInternedPath);
+    return normalizedInternedPath;
```

Comment:
At this PR, shall we return it simply instead of defining `final String`?
```java
- final String normalizedInternedPath = new File(notNormalizedPath).getPath().intern();
- return normalizedInternedPath;
+ return new File(notNormalizedPath).getPath().intern();
```